confbot_description/urdf/confbot.urdf

```diff
@@ -11,19 +11,45 @@
   </material>
 
   <link name="base_link">
+    <inertial>
+      <mass value="5"/>
+      <!--inertia ixx="0.4" ixy="0" ixz="0" iyy="0.4" iyz="0" izz="0.8"/-->
+      <inertia ixx="1" ixy="0" ixz="0" iyy="1" iyz="0" izz="1"/>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+    </inertial>
+    <collision>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <geometry>
+        <box size="1 1 0.1"/>
+        <!--mesh filename="package://confbot_description/meshes/confbot.dae"/-->
+      </geometry>
+    </collision>
     <visual>
       <origin rpy="0 0 0" xyz="0 0 0"/>
       <geometry>
-        <mesh filename="package://confbot_description/meshes/confbot.dae"/>
+        <!--mesh filename="package://confbot_description/meshes/confbot.dae"/-->
+        <mesh filename="file:///media/mikatchou/ROS/ros2/roscon2019/roscon2019_ws/install/share/confbot_description/meshes/confbot.dae"/>
```

Comment:
This should be changed. Not sure why the resource cannot be retrieved when using `package://`

Reply:
By tweaking `GAZEBO_MODEL_PATH` and accepting some additionnal error message in terinal we should be able to work around this. removed in https://github.com/Karsten1987/roscon2018/pull/19/commits/e00874ac34adb066a2e3af9021058b84201d0bf6